Modules/Filtering/ImageFeature/test/itkZeroCrossingBasedEdgeDetectionImageFilterTest.cxx

```diff
@@ -20,35 +20,71 @@
 #include "itkZeroCrossingBasedEdgeDetectionImageFilter.h"
 #include "itkNullImageToImageFilterDriver.hxx"
 #include "itkSimpleFilterWatcher.h"
+#include "itkTestingMacros.h"
 
 int
-itkZeroCrossingBasedEdgeDetectionImageFilterTest(int, char *[])
+itkZeroCrossingBasedEdgeDetectionImageFilterTest(int argc, char * argv[])
 {
-  try
+  if (argc != 3)
   {
-    using ImageType = itk::Image<float, 2>;
-
-    // Set up filter
-    itk::ZeroCrossingBasedEdgeDetectionImageFilter<ImageType, ImageType>::Pointer filter =
-      itk::ZeroCrossingBasedEdgeDetectionImageFilter<ImageType, ImageType>::New();
-
-    itk::SimpleFilterWatcher watcher(filter);
-    filter->SetVariance(1.0f);
-    filter->SetMaximumError(.01f);
-
-    // Run Test
-    itk::Size<2> sz;
-    sz[0] = 100;
-    sz[1] = 100;
-    itk::NullImageToImageFilterDriver<ImageType, ImageType> test1;
-    test1.SetImageSize(sz);
-    test1.SetFilter(filter);
-    test1.Execute();
+    std::cerr << "Missing Parameters." << std::endl;
```

Comment:
Incorrect number of arguments. Even more args than necessary would trigger this.

Reply:
I tend to use the inequality rather than the `<` since I do not see the advantage of the latter unless optional arguments are used.

Reply:
I meant either change condition to `<` or change the error message (for example to `Incorrect number of arguments.`).

Reply:
Now I see the point better and `<` fits better in the error message. Done in 7539a55.